src/providers/ldap/ldap_id.c

```diff
@@ -1476,15 +1484,15 @@ sdap_handle_acct_req_send(TALLOC_CTX *mem_ctx,
 
     case BE_REQ_SUBID_RANGES:
 #ifdef BUILD_SUBID
-        if (!ar->extra_value) {
+        if ((strcasecmp(sdom->dom->provider, "ldap") != 0) &&
+            (strcasecmp(sdom->dom->provider, "ipa") != 0)) {
```

Comment:
Check will not work as expected for a case "IPA-AD trusted subdomain" since "provider" will be "ipa" in this case.